README.md

```diff
@@ -237,43 +237,54 @@ Must specify multiInputOptions with valid entries in the following format:
 
 <details>
   <summary> Code Example </summary>
-    
+
  ```javascript
 prompt({
-	title: "credentials",
-	label: "Please enter username and password",
-	type: "multiInput",
-	multiInputOptions:
-    [{
-        inputAttrs:
-             {
-                type: "email",
-                required: true,
-                placeholder: "email"
-              }
-      },
-      {
-         inputAttrs:
-             {
-                type: "password",
-                placeholder: "password"
-              }
-     }],
-	resizable: true,
-	height: 150,
-	width: 300,
+    title: "credentials",
+    label: "Login Info:",
+    type: "multiInput",
+    multiInputOptions:
+
```

Comment:
I think the opening `[` might be missing

Reply:
fixed in [#0f6c42a](https://github.com/Araxeus/custom-electron-prompt/pull/7/commits/0f6c42a405443c9f103c4fc97b0ec0b0d05c3b64)